libs/pong/get.js

```diff
@@ -8,6 +8,9 @@ const PLACEMENTS = {
   topBanner: 585,
 };
 
+// allow list for client sent keywords
+const KEYWORDS = [];
```

Comment:
Let's rename it to `ALLOWED_KEYWORDS` and remove the comment.

PS: Why do we even extract the keywords from the body, if we filter them all out now?